### PR TITLE
Update logos on /appliance/hardware

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates logos on /appliance/hardware based on [the issue](https://warthogs.atlassian.net/browse/WD-1729)

## QA

- Go to https://ubuntu-com-12609.demos.haus/appliance/hardware and make sure the page matches the 'replacement' images found in the issue

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1729

### CODE REVIEW

**Comment by petesfrench** [comment]
@carkod I will assign these to you, feel free to assign yours to me

**Review by carkod — APPROVED** [review]
Design background for LXD logos seems to have changed? but it might be a "typo". I left a comment.

Also I'm not sure we should wait for design review for this.

**Comment by petesfrench** [comment]
@carkod I saw that, but it wouldn't match the rest of the page. The section below is also `--light` and there is a line break above. I will merge this an address this later when Camille returns

**Pull request merged**
